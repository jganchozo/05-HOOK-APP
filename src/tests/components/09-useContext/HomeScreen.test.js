import { mount } from 'enzyme';
import React from 'react';
import { HomeScreen } from '../../../components/09-useContext/HomeScreen';
import { UserContext } from '../../../components/09-useContext/UserContext';

describe('Pruebas en HomeScreen', () => { 

    const user = {
        name: 'Jose',
        email: 'jganchozo@outlook.com'
    }

    const wrapper = mount(
        <UserContext.Provider value={{
            user
        }}>
            <HomeScreen /> );
        </UserContext.Provider>
    )

    test('should devolver el componente correcte', () => {
        expect(wrapper).toMatchSnapshot();
    });

 });