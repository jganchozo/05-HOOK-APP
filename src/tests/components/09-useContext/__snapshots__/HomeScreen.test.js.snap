// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en HomeScreen should devolver el componente correcte 1`] = `
Array [
  <div>
    <h1>
      HomeScreen
    </h1>
    <hr />
    <pre>
      {
   "name": "Jose",
   "email": "jganchozo@outlook.com"
}
    </pre>
  </div>,
  " );",
]
`;
